src/components/SpiceItem.js

```diff
@@ -3,11 +3,12 @@ import React from 'react'
 class SpiceItem extends React.Component {
 
   render() {
+    console.log(this.props)
     return (
       <div className="spice-item">
-        <img src={null /* replace me with an image url */} alt={"replace me with a TITLE"} />
+        <img src={this.props.spice.image} alt={this.props.spice.title} />
         <div className="details">
-          <h2>{"replace me with a TITLE"}</h2>
+          <h2>{this.props.title}</h2>
           <p>{"replace me with a DESCRIPTION"}</p>
           <em>{"replace me with NOTES"}</em>
```

Comment:
Here, you'd want to use `this.props.spices` to access the spice object and insert the values into your JSX:

```jsx
          <h2>{this.props.spice.title}</h2>
          <p>{this.props.spice.description}</p>
          <em>{this.props.spice.notes}</em>
```

Reply:
ok